### DISCUSSION TITLE
sonoff 4CHPROR3 with the tasmota 10.1.0(sensors) exception

### DISCUSSION THREAD

**Discussion by nbranquinho** [arendst/Tasmota#14770] [Q&A] [answered]
I have the sonoff 4CHPROR3 with the tasmota 10.1.0(sensors) and from time to time it becomes inaccessible. I can not access by http and the entities are also unavailable. The only way I can do it is to restart it through the RF command (I temporarily created a rule for this purpose).

In HA, in the entity ".tasmota_restart_reason" I have the indication of "Exception" and in fact if I consult the information through the "Tasmota Device Manager" I get a red indication of "Exception" but I cannot find information or what caused it the exception.

How can I get more details?
![image](https://user-images.githubusercontent.com/2866871/152982056-667f5488-baf5-4eb0-ab03-754ca46c4550.png)

resultado do STATUS 0
`11:55:37.528 MQT: stat/luzes-cave/STATUS12 = {"StatusSTK":{"Exception":29,"Reason":"Exception","EPC":["4000e1b2","00000000","00000000"],"EXCVADDR":"00000000","DEPC":"00000000","CallChain":["4026e5a4","40294ee0","40101750","4027b31f","4027b313","40268c1f","4026ca0f","4000050c","40000f68","40000f58","4026e9ab","4026e5a4","4026e124","4026e5df","40270558","4027051c","4026e594","4026e568","4026e533","40100d29","4026eaa1","402790bf","40257898","4027edd5","4027ee1a","40256464","40000f49","40000f49","40000f49","40000e19","40105d31"]}}`

**Comment by sfromis** [comment, marked as answer]
Unfortunately, causes of exception 29 can vary greatly, and thus this does not indicate why it happened. One common cause is power issues. Are you powering it via DC?

**Reply by nbranquinho** [reply]
> Unfortunately, causes of exception 29 can vary greatly, and thus this does not indicate why it happened. One common cause is power issues. Are you powering it via DC?

Nop, it's AC220v

**Comment by Jason2866** [comment]
Last try. Erase flash. Flash via serial. Do NOT import old settings via Restore. Configure device from scratch.
If it still generates exceptions it is a hardware defect.
